test_closest (__main__.TestExercice) ... FAIL
test_coordinates (__main__.TestExercice) ... ok
test_linear_values (__main__.TestExercice) ... FAIL

======================================================================
FAIL: test_closest (__main__.TestExercice)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_exercice.py", line 42, in test_closest
    self.assertEqual(
AssertionError: 10 != 3 : Erreur dans le remplacement de caractère

======================================================================
FAIL: test_linear_values (__main__.TestExercice)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test_exercice.py", line 17, in test_linear_values
    self.assertListEqual(
AssertionError: Lists differ: [-1.4, -1.338095238095238, -1.276190476190476[1220 chars] 2.5] != [-1.3, -1.2396825396825397, -1.17936507936507[1225 chars] 2.5]

First differing element 0:
-1.4
-1.3

Diff is 3447 characters long. Set self.maxDiff to None to see it. : Mauvaise identification de la parité de la longueur de la chaine

----------------------------------------------------------------------
Ran 3 tests in 0.042s

FAILED (failures=2)
